File: Python/swapping.py
num1 = int(input("enter a first number: "))
num2 = int(input("enter a second number: "))

print("-------Before swapping-------")
print("first number: ",num1)
print("second number: ",num2)

"""temp = num1
num1 = num2
num2 = temp"""
#without using third variable 
num1 = num1 + num2
num2 = num1 - num2
num1 = num1 - num2

print("-------After swapping--------")
print("first number:",num1,)
print("second number",num2)

File: C/swapping.c
#include <stdio.h>

int main () {
    int num1 , num2;
    
    printf("Enter a first number ");
    scanf("%d",&num1);

    printf("Enter a second number ");
    scanf("%d",&num2);


    printf("------Before Swapping------\n");
    printf("First number %d \n",num1);
    printf("Second number %d \n",num2);

    // Without use third variable

    printf("------After Swapping------\n");
    printf("First number %d \n",num2);
    printf("Second number %d \n",num1);

}